src/main/java/io/github/lucasstarsz/fastj/math/Maths.java

```diff
@@ -136,4 +136,17 @@ public static float magnitude(Point p) {
     public static boolean floatEquals(float a, float b) {
         return Math.abs(a - b) < FloatPrecision;
     }
+
+    /**
+     * Linearly interpolates the value of lerping between x and y, by the interpolation point t.
+     *
+     * @param x The first {@code double} value.
+     * @param y THe second {@code double} value.
+     * @return The lerped value
+     */
+    public static float lerp(float x, float y, float t) {
+        return (1 - t) * x + t * y;
```

Comment:
The same situation from the unit test numbers is apparent here: add an `f` to the end of your 1 there.